### DISCUSSION TITLE
Standup (Friday, May 20th)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#68] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

``` 

**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x

```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Designed workflow for authentication with @CarlosCRG19 
 - Finished company signUp endpoint

**What are your priorities for the next 24 hours?**:
 - Start writing login endpoint

**Blockers**:
 - I am travelling to my hometown ☀️🏝
**Shoutouts**:
 - @CarlosCRG19 for such a productive pair programming sessiok yesterday

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Just some testing on the form.

**What are your priorities for the next 24 hours?**:
 - Custom styling, now for real

**Blockers**:
 - Lots of work
 - Lots of hw

**Shoutouts**:
 - @DavBot02 @paotg123 For receiving me on the database team.

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Chips are working now
 - Re-render loop solved

**What are your priorities for the next 24 hours?**:
 - Work on info handling

**Blockers**:
 - Tired

**Shoutouts**:
 - @BrandonMagana
